common/constants/shared.ts

```diff
@@ -258,7 +258,7 @@ export const VISUALIZATION_ERROR = {
   NO_METRIC: 'Invalid Metric MetaData',
 };
 
-export const S3_DATASOURCE_TYPE = 'S3_DATASOURCE';
+export const S3_DATA_SOURCE_TYPE = 's3glue';
```

Comment:
why s3glue and not glue ?

Reply:
this is for data selector grouping, the s3 data source type returned from backend is s3glue